dstep/translator/MacroDefinition.d

```diff
@@ -579,22 +579,32 @@ string spelling(Expression expression)
     return identifier.spelling;
 }
 
-void translateMacroDefinitionConstant(
+void translateMacroDefinitionAliasOrConst(
     Output output,
     Context context,
     TypedMacroDefinition definition)
 {
     auto expressionContext = ExpressionContext.make(context);
 
-    version (D1)
-        enum formatString = "const %s = %s;";
+    string formatString;
+    auto debraced = definition.definition.expr.debraced;
+
+    if (debraced.peek!TypeIdentifier)
+    {
+        formatString = "alias %s = %s;";
```

Comment:
Not sure if we need to bother but this will only work with D2. @mihails-strasuns?

Reply:
How was it solved in D1? `alias old_name_t new_name_t;` ?

Reply:
Yes.

Reply:
Updated, but I'll leave it without test. No idea how to even test it as the `version (D1)` is compile-time statement...

Reply:
That's fine.